lib/api/trackingManager.dart

```diff
@@ -35,30 +37,27 @@ class TrackingManager {
     switch (response.statusCode) {
       case 200:
       case 204:
-        print("activate sent with success ");
+        Flagship.logger(Level.INFO, ACTIVATE_SUCCESS);
         break;
       default:
-        print("Failed to send activate");
+        Flagship.logger(Level.INFO, ACTIVATE_FAILED);
```

Comment:
Maybe a warning here ?

Reply:
oh yess sound more sense to set a warning